policy/PfsMapper.yaml

```diff
@@ -396,7 +396,7 @@ datasets:
     - raw
 
   dmQaResidualImage:
-    template: 'detectorMapQA/%(dateObs)s/v%(visit)06d/dmQaResidualImage-%(visit)06d-%(arm)s%(spectrograph)d.pdf'
+    template: 'detectorMapQA/%(dateObs)s/v%(visit)06d/dmQaResidualImage-%(visit)06d-%(arm)s%(spectrograph)d.png'
     python: matplotlib.backends.backend_pdf.PdfPages
```

Comment:
Despite the fact that this works fine (since we can't round-trip things with `storage: MatplotlibStorage`) I'm guessing we don't want to save it as a `png` and still have this `python` object here.  

LSST seems to be able to store png plots (as per [this discussion](https://community.lsst.org/t/exporting-an-image-with-butler/5872)) but I can't find any examples.